rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -73,6 +73,7 @@ rabbitmq_ulimit: 65535
 # since memory utilization is primarily driven by the objects placed in memcached.
 memcached_connections: 16384
 
+#TODO (evrardjp): Remove this when ocata
 # Keystone overrides
 keystone_token_driver: "sql"
```

Comment:
Ah, so I see what's happening here. Why don't we make OSA smarter instead of carrying this? In OSA there's a note to use the sql token driver if you're using sql for token storage, so why don't we adjust it so that this happens automatically instead of requiring the user to set it?